Project/Assets/Scripts/CombatManager.cs

```diff
@@ -1,10 +1,14 @@
 ﻿using UnityEngine;
 using System.Collections;
 
+//Combat Manager
```

Comment:
The usefulness of this class is:
Allows us to filter damage transactions at a global level
Allows us to log all damage transactions.

Right now this class is a little thin, however, If we design around its useful points, I think we can develop a system that provides greater runtime visibility and functionality